integration/matchers/cluster.go

```diff
@@ -0,0 +1,32 @@
+package matchers
+
+import (
+	"strings"
+
+	"github.com/weaveworks/eksctl/integration/runner"
+
+	. "github.com/onsi/gomega"
+)
+
+type GetClusterOutput struct {
+	ClusterName   string
+	Region        string
+	EksctlCreated string
+}
+
+// AssertContainsCluster asserts that the output of the specified command contains the specified cluster
+func AssertContainsCluster(cmd runner.Cmd, out GetClusterOutput) {
+	Expect(cmd).To(runner.RunSuccessfullyWithOutputStringLines(
```

Comment:
Intentionally not using a dot import because runner.Cmd is also being used here.